packages/server-reason-react/server-reason-react.0.1.0/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "Rendering React components on the server natively"
+maintainer: ["David Sancho <dsnxmoreno@gmail.com>"]
+authors: ["David Sancho <dsnxmoreno@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/ml-in-barcelona/server-reason-react"
+bug-reports: "https://github.com/ml-in-barcelona/server-reason-react/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml" {>= "5.0.0"}
+  "reason" {>= "3.10.0"}
+  "melange" {>= "3.0.0"}
+  "ppxlib" {> "0.23.0" & <= "0.31.0"}
+  "pcre" {<= "7.5.0"}
+  "promise" {>= "1.1.2"}
+  "lwt" {>= "5.6.0"}
+  "lwt_ppx" {>= "2.1.0"}
+  "uri" {>= "4.2.0"}
+  "alcotest" {with-test}
+  "alcotest-lwt" {with-test}
+  "fmt" {with-test}
+  "merlin" {with-test}
+  "odoc" {with-doc}
+  "ocamlformat" {= "0.26.1" & with-test}
+  "ocaml-lsp-server" {with-test}
+  "tiny_httpd" {with-test}
+]
+dev-repo: "git+https://github.com/ml-in-barcelona/server-reason-react.git"
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@new-doc" {with-doc}
+  ]
+]
+url {
```

Comment:
```suggestion
flags: verbose
url {
```
This should show us what is going on

Reply:
The build failed in fsb in 5.1, but still didn't gave us many info. The only special part (from my pov) on this repo is either profile or a dependency that breaks silently (pcre?)